### PR TITLE
Add details on oracle bytestream reading

### PR DESCRIPTION
**Pull request by mbthiery**

### CODE REVIEW

File: docs/oracles/oracle-data.mdx [hunk 1]
```diff
@@ -106,8 +106,19 @@ patterns.
...
 In order to store the massive amount of data generated by the Oracles, the output is stored as
-zipped bytestreams. In order to 'unpack' this data, you'll need the associated protobuf from
-[github.com/helium/proto](https://github.com/helium/proto).
+zipped bytestreams. In order to 'unpack' this data, you'll first need the associated protobuf from
+[github.com/helium/proto](https://github.com/helium/proto). You also can use the mappings above to
+find the correct protobuf definition.
+
+Once uncompressed, the file is a series of 4 byte prefix length encoded binary followed by the
+associated message. More concretely, you'll need to repeatedly read the first 4 bytes and decode
+that to a u32. Then use that u32 number as the number of subsequent bytes to read. Decode those
```
Comment (madninja) [inline docs/oracles/oracle-data.mdx]:
```suggestion
that as a big endian u32. Then use that u32 number as the number of subsequent bytes to read. Decode those
```

File: docs/oracles/oracle-data.mdx [hunk 2]
```diff
@@ -106,8 +106,19 @@ patterns.
...
+subsequent bytes using the associated proto definition. Repeat this process until you've read the
+entire file.
+
+Alternatively you can adapt
+[this sample oracle observer repo](https://github.com/dewi-alliance/oracle-observer-sample) for your
```
Comment (deasydoesit) [inline docs/oracles/oracle-data.mdx]: Can we link to the lambda code?

**Review by madninja — COMMENTED** [review]
just one endianness comment for the 4 byte prefix

**Comment by deasydoesit** [comment]
This is probably sufficient for our technical audience. It would be going above and beyond to give inline examples in Rust and Node for how to decode files but I leave it up to you to decide.

**Review by madninja — DISMISSED** [review]

File: docs/oracles/oracle-data.mdx (hunk 2 above)
Comment (mbthiery) [reply docs/oracles/oracle-data.mdx]: Do you mean link the lambda folder instead of the repo as a whole?

Comment (deasydoesit) [reply docs/oracles/oracle-data.mdx]: I meant the lambda code e.g., `main.rs`, but the lambda dir works as well

**Review by deasydoesit — APPROVED** [review]

**Pull request merged**